client/internal/templates/pkce-auth-msg.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+<head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <style>
+        body {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            min-height: 100vh;
+            margin: 0;
+            background: #f7f8f9;
+            font-family: sans-serif, Arial, Tahoma;
+        }
+        .container {
+               width: 100%;
+    background: white;
+    border: 1px solid #e8e9ea;
+    text-align: center;
+    padding: 20px;
+    padding-bottom: 50px;
+    max-width: 550px;
+    margin: 0 10px;
+        }
+        .logo {
+            height: 80px;
+            border-bottom: 1px solid #e8e9ea;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+        .logo img {
+            width: 130px;
+        }
+        .content {
+            font-size: 13px;
+            color: #525252;
+            line-height: 18px;
+            padding: 10px 0;
+        }
+        .content div {
+            font-size: 18px;
+            line-height: normal;
+            margin-bottom: 5px;
+            color: black;
+        }
+        .cta-button {
+            color: white;
+            text-decoration: none;
+            font-size: 18px;
+            background: #f68330;
+            padding: 16px 26px;
+            display: inline-block;
+            border-radius: 5px;
+        }
+        .success-icon {
+            width: 40px;
+            height: 40px;
+            background: url('URL_OF_SUCCESS_ICON') no-repeat center center;
+            background-size: cover;
+            margin-bottom: 10px;
+        }
+        .signature {
+            font-size: 12px;
+            color: #bbb;
+            margin-top: 15px;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <div class="logo">
+            <a href="https://netbird.io" target="_blank">
+                <img src="https://img.mailinblue.com/6211297/images/content_library/original/64bd4ce82e1ea753e439b6a2.png" class="CToWUd" data-bit="iit">
```

Comment:
It is a logo, a small image, so it's better if we will use `src="data:image/jpeg;charset=utf-8;base64,<DATA>"` notation for the image tag instead of referencing it to an internet resource.

Reply:
converting an image into base64 significantly increase the size of the file.